core/src/components/item/test/legacy/disabled/index.html

```diff
@@ -86,13 +81,11 @@
           </ion-list-header>
 
           <ion-item>
-            <ion-checkbox disabled slot="start" legacy="true"></ion-checkbox>
-            <ion-input placeholder="Disabled Checkbox w/ Input" legacy="true"></ion-input>
+            <ion-input placeholder="Input" legacy="true"></ion-input>
```

Comment:
Same thoughts as the comment above.

Reply:
https://github.com/ionic-team/ionic-framework/pull/29020/commits/19af7b5b5bcb2bb49bcc7186cd122119c3c895a3